src/database/DataTypes/Location.js

```diff
@@ -73,6 +73,24 @@ export class Location extends Realm.Object {
   get coldConsecutiveBreachConfig() {
     return this.breachConfigs.filtered("type == 'COLD_CONSECUTIVE'")[0];
   }
+
+  get isInHotBreach() {
+    return !!this.breaches.filtered('type == "HOT_CONSECUTIVE" && endTimestamp == null')[0];
+  }
+
+  get isInColdBreach() {
+    return !!this.breaches.filtered('type == "COLD_CONSECUTIVE" && endTimestamp == null')[0];
+  }
+
+  // Sensor to location should be a one to one relationship at all times. Pick the first and
+  // make the assumption it is the correct one.
```

Comment:
(comment) I truuuuly believe one day annoyingly soon this assumption will be wrong

Reply:
Bit confused given we collectively (Andrei, you and I) decided that a location and sensor would be a one to one! 🤷